contracts/src/v0.8/automation/upkeeps/LinkAvailableBalanceMonitor.sol

```diff
@@ -128,32 +161,32 @@ contract LinkAvailableBalanceMonitor is ConfirmedOwner, Pausable, KeeperCompatib
   }
 
   /**
-   * @notice Gets a list of proxies that are underfunded, up to the MAX_PERFORM size
+   * @notice Gets a list of proxies that are underfunded, up to the s_maxPerform size
    * @dev the function starts at a random index in the list to avoid biasing the first
    * addresses in the list over latter ones.
-   * @dev the function will check at most MAX_CHECK proxies in a single call
-   * @dev the function returns a list with a max length of MAX_PERFORM
+   * @dev the function will check at most s_maxCheck proxies in a single call
+   * @dev the function returns a list with a max length of s_maxPerform
    * @return list of target addresses which are underfunded
    */
   function sampleUnderfundedAddresses() public view returns (address[] memory) {
     uint256 numTargets = s_watchList.length();
     uint256 numChecked = 0;
```

Comment:
I know it's not your code but let's just move this into the for loop declaration

Reply:
Revisited the for and adjusted appropiatedly.